src/modules/RA/Picture/components/PictureFields.js

```diff
@@ -30,6 +30,13 @@ const styles = {
   },
 };
 
+const weatherCondition = [
```

Comment:
Maybe adding plural to variable name, `weatherConditions` to be more explicit. Also, french should not be wrote directly in the code base, translated content is prefered.

Since those data will be hard coded here, maybe it is preferable to send more human readable codename instead of numeric id to backend? Something like `good_weather`, `lightly_covered` etc